csrc/flash_attn/src/flash_fwd_kernel.h

```diff
@@ -1410,35 +1411,37 @@ __forceinline__ __device__ void compute_attn_1rowblock_flashmask(const Params &p
             // if (cute::thread0()) { print(idx_row.layout()); print(stride<1>(idx_row)); printf("stride = %d \n", get<1>(stride<1>(idx_row))); }
             // I can't get the stride from idx_row
             if (true/*Is_flashmask*/ && (!enable_mask_bypass ||
```

Comment:
remove the boolean literal `true`, or add following definition: `constexpr bool Is_flashmask = true;`